index.js

```diff
@@ -1,71 +1,202 @@
 'use strict'
 
-var os = require('os')
-var app = require('app')
+var menubar = require('menubar')
 var BrowserWindow = require('browser-window')
+var app = require('app')
+
+var fs = require('fs')
+
+var os = require('os')
 var ipfsd = require('ipfsd-ctl')
-var multiaddr = require('multiaddr')
-var Tray = require('tray')
-var menu = require('menu')
-var argv = require('minimist')(process.argv.slice(1))
+var ipc = require('ipc')
+var open = require('open')
 
+var multiaddr = require('multiaddr')
 var WEBUIPATH = '/webui'
 
 var LOGO = __dirname + '/node_modules/ipfs-logo/ipfs-logo-256-ice.png'
 var TRAY_ICON = (os.platform() !== 'darwin' ? LOGO
                  : __dirname + '/node_modules/ipfs-logo/platform-icons/osx-menu-bar.png')
 
-// keep references around for gc purposes
-var mainWindow = null
-var mainTray = null
+// only place where app is used directly
+var IPFS_PATH_FILE = app.getDataPath() + '/ipfs-electron-app-node-path'
 
-var wizard = function (err) {
-  mainWindow = new BrowserWindow({icon: LOGO, width: 800, height: 600})
-  mainWindow.loadUrl('file://' + __dirname + '/wizard/wizard.html')
+var MENU_WIDTH = 240
+
+var mainWindow
+var Ipfs
+
+var error = function (err) {
+  mainWindow = new BrowserWindow({icon: LOGO,
+                                  'auto-hide-menu-bar': true,
+                                  width: 800,
+                                  height: 600})
+  mainWindow.loadUrl('file://' + __dirname + '/html/help.html')
   mainWindow.webContents.on('did-finish-load', function () {
+    console.log('in loaded')
```

Comment:
We should probably add a debugger module. [debug](https://www.npmjs.com/package/debug) is always my go-to, but I have no real preference.

(though this line may not even be needed for debugging)